atspi-common/src/events/mod.rs

```diff
@@ -18,12 +18,43 @@ pub mod window;
 // specific signal types with TryFrom implementations. See crate::[`identify`]
 //  EVENT_LISTENER_SIGNATURE is a type signature used to notify when events are registered or deregistered.
 //  CACHE_ADD_SIGNATURE and *_REMOVE have very different types
-pub const ATSPI_EVENT_SIGNATURE: Signature<'_> =
-	Signature::from_static_str_unchecked("(siiva{sv})");
-pub const QSPI_EVENT_SIGNATURE: Signature<'_> = Signature::from_static_str_unchecked("(siiv(so))");
-pub const EVENT_LISTENER_SIGNATURE: Signature<'_> = Signature::from_static_str_unchecked("(ss)");
-pub const CACHE_ADD_SIGNATURE: Signature<'_> =
-	Signature::from_static_str_unchecked("((so)(so)(so)iiassusau)");
+// Same as "(siiva{sv})"
```

Comment:
We already have `EventBodyOwned::SIGNATURE` right? And the same for the Qt variant? I think removing these as part of the migration make sense because the new API to build signatures is very hard to read.